2024-02-26 01:12:50 error: uncaughtException: Could not perform discovery. No routing servers available. Known routing table: RoutingTable[database=default database, expirationTime=0, currentTime=1708938770921, routers=[], readers=[], writers=[]]
Neo4jError: Could not perform discovery. No routing servers available. Known routing table: RoutingTable[database=default database, expirationTime=0, currentTime=1708938770921, routers=[], readers=[], writers=[]]
    at new Neo4jError (/Users/gk/Documents/projects/tripify/node_modules/neo4j-driver-core/lib/error.js:77:16)
    at newError (/Users/gk/Documents/projects/tripify/node_modules/neo4j-driver-core/lib/error.js:113:12)
    at RoutingConnectionProvider.<anonymous> (/Users/gk/Documents/projects/tripify/node_modules/neo4j-driver-bolt-connection/lib/connection-provider/connection-provider-routing.js:704:68)
    at step (/Users/gk/Documents/projects/tripify/node_modules/neo4j-driver-bolt-connection/lib/connection-provider/connection-provider-routing.js:100:23)
    at Object.next (/Users/gk/Documents/projects/tripify/node_modules/neo4j-driver-bolt-connection/lib/connection-provider/connection-provider-routing.js:81:53)
    at /Users/gk/Documents/projects/tripify/node_modules/neo4j-driver-bolt-connection/lib/connection-provider/connection-provider-routing.js:75:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/gk/Documents/projects/tripify/node_modules/neo4j-driver-bolt-connection/lib/connection-provider/connection-provider-routing.js:71:12)
    at RoutingConnectionProvider._applyRoutingTableIfPossible (/Users/gk/Documents/projects/tripify/node_modules/neo4j-driver-bolt-connection/lib/connection-provider/connection-provider-routing.js:698:16)
    at RoutingConnectionProvider.<anonymous> (/Users/gk/Documents/projects/tripify/node_modules/neo4j-driver-bolt-connection/lib/connection-provider/connection-provider-routing.js:508:55)
2024-02-26 01:12:50 error: uncaughtException: Could not perform discovery. No routing servers available. Known routing table: RoutingTable[database=default database, expirationTime=0, currentTime=1708938770926, routers=[], readers=[], writers=[]]
Neo4jError: Could not perform discovery. No routing servers available. Known routing table: RoutingTable[database=default database, expirationTime=0, currentTime=1708938770926, routers=[], readers=[], writers=[]]
    at new Neo4jError (/Users/gk/Documents/projects/tripify/node_modules/neo4j-driver-core/lib/error.js:77:16)
    at newError (/Users/gk/Documents/projects/tripify/node_modules/neo4j-driver-core/lib/error.js:113:12)
    at RoutingConnectionProvider.<anonymous> (/Users/gk/Documents/projects/tripify/node_modules/neo4j-driver-bolt-connection/lib/connection-provider/connection-provider-routing.js:704:68)
    at step (/Users/gk/Documents/projects/tripify/node_modules/neo4j-driver-bolt-connection/lib/connection-provider/connection-provider-routing.js:100:23)
    at Object.next (/Users/gk/Documents/projects/tripify/node_modules/neo4j-driver-bolt-connection/lib/connection-provider/connection-provider-routing.js:81:53)
    at /Users/gk/Documents/projects/tripify/node_modules/neo4j-driver-bolt-connection/lib/connection-provider/connection-provider-routing.js:75:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/gk/Documents/projects/tripify/node_modules/neo4j-driver-bolt-connection/lib/connection-provider/connection-provider-routing.js:71:12)
    at RoutingConnectionProvider._applyRoutingTableIfPossible (/Users/gk/Documents/projects/tripify/node_modules/neo4j-driver-bolt-connection/lib/connection-provider/connection-provider-routing.js:698:16)
    at RoutingConnectionProvider.<anonymous> (/Users/gk/Documents/projects/tripify/node_modules/neo4j-driver-bolt-connection/lib/connection-provider/connection-provider-routing.js:508:55)
